migration/identity-migration/src/main/java/io/camunda/migration/identity/config/sm/StaticEntities.java

```diff
@@ -310,13 +313,7 @@ public static List<CreateAuthorizationRequest> getAuthorizationsByAudience(
                     AuthorizationResourceType.DECISION_DEFINITION,
                     Set.of(
                         PermissionType.CREATE_DECISION_INSTANCE,
-                        PermissionType.DELETE_DECISION_INSTANCE)),
-                new CreateAuthorizationRequest(
-                    ownerId,
-                    ownerType,
-                    "*",
-                    AuthorizationResourceType.DECISION_REQUIREMENTS_DEFINITION,
-                    Set.of(PermissionType.UPDATE, PermissionType.DELETE))));
```

Comment:
💭 zeebe:write already has the fitting resource permissions create, and all deletes but no read